added one-time donation screen but had trouble with form validation

malawi-mike opened the pull request:
please try to fix/add form validation. based my work off of the first form in here: view-source:http://demo.geekslabs.com/materialize-v1.0/form-validation.html

jdepree commented:
Whenever I click a donate button, I'm getting "Page not Found" for this page: one_time_payment_modal.php - any idea what happened here?

malawi-mike commented:
Replaced the modal with a page view. Need to check whether I deleted the modal file. I've linked the payment flow on my local version. Need to push that up.

Mike Buckler
CEO, Village X Org
mike@villagex.org
202-360-9931

Sent from my iPhone

> On Aug 28, 2017, at 1:04 AM, Jeff DePree <notifications@github.com> wrote:
> 
> Whenever I click a donate button, I'm getting "Page not Found" for this page: one_time_payment_modal.php - any idea what happened here?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

jdepree commented:
Check the diff - you have something called temp/payment_modal.php and one_time_payment_view.php

jdepree commented:
it's more standard to name branches according to their function rather than by Malawian village names.

malawi-mike commented:
I think I changed the name to one_time_payment_view. I think I've updated
the links on my local copy. Sorry!

On Mon, Aug 28, 2017 at 1:04 AM, Jeff DePree <notifications@github.com>
wrote:

> Whenever I click a donate button, I'm getting "Page not Found" for this
> page: one_time_payment_modal.php - any idea what happened here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Village-X-Org/Site-v2/pull/32#issuecomment-325260759>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AcyICtu_LQ_OYtMTM1fU72E3qOkhyg9Qks5sckpVgaJpZM4PD6gM>
> .
>



-- 
Mike Buckler, CEO & General Counsel
Village X Org <http://www.villagexapp.com>
mike@villagex.org
202-360-9931
<http://www.villagexapp.com>
